gettsim/demographic_vars.py

```diff
@@ -327,3 +327,81 @@ def rentner_in_hh(hh_id: IntSeries, rentner: BoolSeries) -> BoolSeries:
     BoolSeries indicating pensioner in household.
     """
     return rentner.groupby(hh_id).any()
+
+
+def steuerklasse(
+    tu_id: IntSeries,
+    gemeinsam_veranlagt: BoolSeries,
+    alleinerziehend: BoolSeries,
+    bruttolohn_m: FloatSeries,
+    kind: BoolSeries,
+    eink_st_params: dict,
+    eink_st_abzuege_params: dict,
+) -> IntSeries:
+    """ Determine Lohnsteuerklassen (also called 'tax brackets')
+    They determine the basic allowance for the withdrawal tax
+
+    1: Single
+    2: Single Parent
+    3: One spouse in married couple who receives allowance of both partners.
+       Only in case of Single-Earner Households
+    4: Both spouses receive their individual allowance
+    5: Other spouse in couple who receives no allowance
+    6: Additional Job...not modelled yet, as we do not
+    distinguish between different jobs
+
+    Parameters
+    ----------
+    tu_id: Tax Unit Identifier
```

Comment:
List of parameters should be in the same style as in other docstrings (e.g. just refer to the respective function or input variable)